### DISCUSSION TITLE
Location of the logs

### DISCUSSION THREAD

**Discussion by TrueTheos** [telekom-security/tpotce#992] [Q&A]
Hello there,

I've logged to honeypot via ssh and I am trying to find logs but with no success. 
Where are they located?

**Comment by TrueTheos** [comment]
Bumping my question.

**Reply by ehnwebmaster** [reply]
Search in Folder

> /data

Structure:
/data/name/log

Where name is the name of honeypot

For example:

> /data/cowrie/log

Clean the logs:

https://github.com/telekom-security/tpotce/blob/master/bin/clean.sh

Rotate logs:

https://github.com/telekom-security/tpotce/discussions/1023
